connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -271,9 +271,7 @@ message Deduplicate {
 
 // A relation that does not need to be qualified by name.
 message LocalRelation {
-  // (Optional) A list qualified attributes.
-  repeated Expression.QualifiedAttribute attributes = 1;
-  // TODO: support local data.
+  bytes data = 1;
```

Comment:
Please add a comment mentioning that the data is stored as arrow IPC message streams and that since the ipc streams contain the schema we don't need to qualify it.